frontend/test/metabase-visual/dashboard/parameters-widget.cy.spec.js

```diff
@@ -0,0 +1,62 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATASET;
+
+const questionDetails = {
+  query: { "source-table": PRODUCTS_ID },
+};
+
+const filter = {
+  name: "Category",
+  slug: "category",
+  id: "ad1c877e",
+  type: "category",
+};
+
+const filters = new Array(12).fill(filter);
```

Comment:
I like this idea, but it might need some tweaking, depending on your end goal.
We're filling this array 12x with absolutely the same filter (because of the hard coded `id` key). It then creates 12 identical filters, which is not even possible to achieve IRL.

Maybe you intentionally wanted to "hack" it this way simply to populate Ui with 12 filters (because we don't actually care about their functionality - we just want to make sure they remain sticky)?

Reply:
@nemanjaglumac 

Yes, as you point out, the idea was just to populate with any 12 filters and test the behaviour of their container.

Happy to change this and follow your instructions if you think we should be specific about these filters.

Reply:
> Happy to change this and follow your instructions if you think we should be specific about these filters.

Nah, this is more than good for this use case. Just wanted to check the intent behind it.
Btw, in that case you can even remove the part where you connect the filter to the card. Still need to make that card tall enough, though. This is optional - don't waste time if you don't want to.